src/main/java/pl/wedding/management/service/weddinghall/model/AddressHall.java

```diff
@@ -0,0 +1,16 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+@Getter
```

Comment:
@Value already provides @Getter
You don't need to add private modifier, as @Value already adds: `private final` as well.